### ISSUE TITLE
Too much logging

### ISSUE THREAD

**Issue by brettviren** [WireCell/wire-cell-sigproc#12]
We should reduce the amount of info sent to stdout/stderr.  Some verbosity in configuration is okay as it's one time but components should not produce much output each event and this output should somehow be useful and non-repetative.  Here are some examples that should be removed.  Not all are part of sigproc, but most are.

```
CookedFrameSource: got 12480 raw::RawDigit objects
	nticks=6000 setting to 6000
WCT: executing 1 apps:
WCT: executing app: Pgrapher
```

```

FrameFanout: tagging frame: orig0
FrameFanout: tagging frame: orig1
FrameFanout: tagging frame: orig2
FrameFanout: tagging frame: orig3
FrameFanout: tagging frame: orig4
FrameFanout: tagging frame: orig5
```
and the other end:

```
FrameFanin: tagging frame: framefanin
FrameFanin: tagging traces: gauss0
FrameFanin: tagging traces: threshold0
FrameFanin: tagging traces: wiener0
FrameFanin: tagging traces: gauss1
FrameFanin: tagging traces: threshold1
FrameFanin: tagging traces: wiener1
FrameFanin: tagging traces: gauss2
FrameFanin: tagging traces: threshold2
FrameFanin: tagging traces: wiener2
FrameFanin: tagging traces: gauss3
FrameFanin: tagging traces: threshold3
FrameFanin: tagging traces: wiener3
FrameFanin: tagging traces: gauss4
FrameFanin: tagging traces: threshold4
FrameFanin: tagging traces: wiener4
FrameFanin: tagging traces: gauss5
FrameFanin: tagging traces: threshold5
FrameFanin: tagging traces: wiener5
```

```
ChannelSelector: see frame: 1462 no tags, whole frame
```

```
OmnibusSigProc: plane index: 0 configured with 0 bad regions
```
etc for other planes.

```
OmnibusSigProc: produce 5120 traces
	2560 wiener1 
	2560 gauss1 
```

```
Retagger: merge: wiener0 -> wiener with 2560 / 15360
Retagger: merge: wiener1 -> wiener with 5120 / 15360
Retagger: merge: wiener2 -> wiener with 7680 / 15360
Retagger: merge: wiener3 -> wiener with 10240 / 15360
Retagger: merge: wiener4 -> wiener with 12800 / 15360
Retagger: merge: wiener5 -> wiener with 15360 / 15360
Retagger: tagging trace set: wiener with 15360 traces, 0 summary
Retagger: merge: gauss0 -> gauss with 2560 / 15360
Retagger: merge: gauss1 -> gauss with 5120 / 15360
Retagger: merge: gauss2 -> gauss with 7680 / 15360
Retagger: merge: gauss3 -> gauss with 10240 / 15360
Retagger: merge: gauss4 -> gauss with 12800 / 15360
Retagger: merge: gauss5 -> gauss with 15360 / 15360
Retagger: tagging trace set: gauss with 15360 traces, 0 summary
```

```
Gen::Frame: #1462 @0.00028 with 30720 traces
	frame tags:[retagger]
	trace tags:[gauss, wiener]
Gen::DumpFrames sees EOS
```
and probably also this final
```
wclsFrameSaver: saving frame to art::Event, frame has trace tags:[ gauss wiener ], and frame tags:[ retagger ]
wclsFrameSaver: saving recob::Wires tagged "gauss"
wclsFrameSaver: saving recob::Wires tagged "wiener"
```

So, basically everything that doesn't seem like a warning....

**Comment by brettviren** [comment]
Much of these have been removed.
